src/Sidebar/Sidebar.js

```diff
@@ -3,7 +3,11 @@ import "./Sidebar.css";
 const Sidebar = () => {
   return (
     <div className="sidebar-main__container">
-        <div></div>
+        <div>Hello</div>
+        <p>Hello</p>
+        <p>Hello</p>
+        <p>Hello</p>
+        <p>Hello</p>
```

Comment:
I appreciate that this is a placeholder component - try to use terms that will represent what they will eventually be